### ISSUE TITLE
[MAS1.4.3] [Visual Requirements-Open bot dialog] Luminosity Ratio is less than 4.5:1 “Learn more “ link.

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1729]
**Actual Result:**
Luminosity Ratio is less than 4.5:1 “Learn more “link present in the open bot dialog 

**Expected Result:**
Luminosity Ratio should be greater than or equal to 4.5:1 for “Learn more “link

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened
3.Navigate to the open bot control and select it 
4.Open bot dialog gest opened 
5.Navigate to the Learn more link 
6. Open 'Colour Contrast Analyser' application.
7. Verify whether Luminosity Ratio is less than 4.5:1 for Learn more link or not 

**User Impact:**
Users with low vision or colour blindness needs that difference between contrast of link is always high.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator
Tool: Colour contrast analyser 

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)
![MAS1 4 3_Open bot dialog](https://user-images.githubusercontent.com/53518994/63157190-05398980-c034-11e9-8af1-9805f8282aba.png)
